manager/integration/tests/common.py

```diff
@@ -3029,7 +3039,9 @@ def get_k8s_zone_label():
 
     k8s_ver_minor = k8s_ver_data.minor
 
-    if int(k8s_ver_minor) >= 17:
+    # minor version is not always integer convertable:
+    #   ValueError: invalid literal for int() with base 10: '20+'
```

Comment:
```
    # k8s_ver_minor no needs to be an int
    # it could be "24+" in eks
    if int(re.sub('\\D', '', k8s_ver_minor)) >= 17:
```

Ref: https://github.com/longhorn/longhorn-tests/pull/1255/files#diff-a2154691ec50c6925b7f385cf30c42f33b2a3c935192c894bfe4b32fe3ee49dcR3034-R3036